Added a GIF and Updated README.md

ShruAgarwal opened the pull request:
Fixing issue #1  ---
- Created and added an animated GIF explaining the working of twrm.io website. (`demo.gif`)
- Made changes in the `README.md` about the usage of the website and it's working along with the demo file.

Open for feedback on this PR 😁

ShruAgarwal committed:
Working of twrm.io

ShruAgarwal committed:
Updated a demo of twrm.io

vsnthdev commented on README.md:
```diff
@@ -29,7 +29,23 @@
+If you want to *@mention* someone, or use a *link*, or even use a *#hashtag* in your tweets without notifying anyone, then **twrm** is here to help you out!  
+
+*Here's how it works : 👇*
+1. Go to  [twrm.io](https://twrm.io/) and use the search box to @mention the twitter handle of the person (from whom you don't want to get notified!)
```
Replace "search box" with "text box" since we're not searching anything with that box, we don't call it a search box.

vsnthdev commented on README.md:
```diff
@@ -29,7 +29,23 @@
+
+*Here's how it works : 👇*
+1. Go to  [twrm.io](https://twrm.io/) and use the search box to @mention the twitter handle of the person (from whom you don't want to get notified!)
+2. After you mention a valid person in the search box, you can see a blue tick aside it. ☑
```
Use ✅ instead of ☑ since renders correctly on almost all devices.

vsnthdev commented:
Also, the GIF shows only after our tool, it would be really amazing if we can follow this process 👇
1. Normally mention a user in tweet editor
2. Now type that mention in twrm.io's magic box ✨
3. Go back to the tweet editor and paste them below the original mention.

ShruAgarwal committed:
Delete demo.gif

ShruAgarwal committed:
Updated with relevant changes

ShruAgarwal commented:
Thanks for your feedback! Will make the changes as much as I can!

ShruAgarwal committed:
Update README.md

ShruAgarwal commented:
Hi @vsnthdev 👋
I've changed the GIF as you asked for.
I've deleted the instructions for using *twrm* from `README.md`  as I think you know your project much better than I can. 
It would be better if you make changes on 'How to use twrm.io' in `README.md`!

Let me know whether the GIF is OK or not...

vsnthdev commented:
Thank you so much ✨ @ShruAgarwal 👏

ShruAgarwal commented:
Thanks for merging it! 😄

Will contribute again if I find something interesting!

On Fri, Mar 18, 2022, 18:42 Vasanth Srivatsa ***@***.***>
wrote:

> Thank you so much ✨ @ShruAgarwal <https://github.com/ShruAgarwal> 👏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vsnthdev/twrm.io/pull/7#issuecomment-1072396772>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATXZWRNCMBYT7IAVA4PAE23VAR6LNANCNFSM5QYNYHEA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
